mapserver/planwise.map

```diff
@@ -30,11 +26,11 @@ MAP
   # WMS requests performed by mapcache
   WEB
     METADATA
-      WMS_TITLE "Kenya"
+      WMS_TITLE "Planwise"
       WMS_ENABLE_REQUEST "*"
       WMS_FORMAT "image/png"
       WMS_SRS "EPSG:3857"
-      WMS_ONLINERESOURCE "/mapserv?map=/etc/mapserver/kenya.map&"
+      WMS_ONLINERESOURCE "/mapserv?map=/etc/mapserver/planwise.map&"
```

Comment:
I don't think the parameter here is necessary. AFAIK this is for Mapserver to generate URLs for the GetCapabilities and other similar requests. And since the mapfile is set by an environment variable this should not be necessary.